tests/threshold_sign.rs

```diff
@@ -124,24 +156,20 @@ proptest! {
     })]
 
     #[test]
-    #[allow(clippy::unnecessary_operation)]
     fn test_threshold_sign_random_silent_200_samples(seed in gen_seed()) {
-        do_test_threshold_sign_random_silent_200_samples(seed)
+        let new_adversary = || ReorderingAdversary::new();
+        test_coin_different_sizes(new_adversary, 200, seed);
     }
 
     #[test]
-    #[allow(clippy::unnecessary_operation)]
     fn test_threshold_sign_first_silent_50_samples(seed in gen_seed()) {
```

Comment:
This function should probably now be called `do_test_coin_random_silent_50_samples`.